### ISSUE TITLE
Favicon.ico not present error when trying to run Vanilla App

### ISSUE THREAD

**Issue by alallier** [kethinov/mkroosevelt#16]
It seems that when I make a vanilla app with mkroosevelt and try and run it, it errors and crashes because of the favicon not being present.

**Comment by kethinov** [comment]
Not able to reproduce this on mkroosevelt 0.1.43 and node 5.2.0 on OS X.

What version / OS combination are you using?

**Comment by alallier** [comment]
Windows 7 and Node version 5.2.0

**Comment by alallier** [comment]
I will test this on my Windows 10 machine and see what happens

**Comment by kethinov** [comment]
*dusts off Win7 VM*

**Comment by alallier** [comment]
Also was able to reproduce this on Windows 10. The file actually isn't present, I can create a dummy file named favicon.ico and it will run great

**Comment by kethinov** [comment]
*puts Win7 VM back in cold storage, boots Win10 VM that is already set up*

**Comment by alallier** [comment]
Haha sorry, I should have tested a bit before making you run around like a chicken with his head cut off

**Comment by kethinov** [comment]
I still can't reproduce it. I'm using Edge. What browser were you using?

**Comment by alallier** [comment]
Chrome, but it's not the browser the server crashes because the file actually isn't there when making the app.

**Comment by kethinov** [comment]
Different browsers will request favicons different ways. That's why I felt it was relevant.

I still can't reproduce this on Win10, Node 5.2.0, freshly created app with mkroosevelt 0.1.43, visiting the homepage with either Edge or Chrome in either dev mode or prod.

**Comment by alallier** [comment]
Okay yeah but when I create the vanilla app and check the path /statics/images there is no actual file named favicon.ico that is made from the command line tool it seems to be missing. It's possible it's some dev environment issue of mine, I'll mess with it more and see what I can dig up, if anything at all

**Comment by kethinov** [comment]
It's definitely being created on mine. If you figure it out, lemme know. Could be an obscure bug that we need to fix, or possibly some unstated premise I'm operating from that needs to be properly documented.

**Comment by alallier** [comment]
Will do

**Comment by alallier** [comment]
Well I traced it back to the global install location, sample app not even having the favicon, so when I created the new app it had no favicon to copy. It must have been some issue when upgrading node version or just something screwy in general. I ended up deleted mkroosevelt entirely and re-installing and that seemed to do the trick tested this both on Windows 7 and 10. With that said I am closing the issue, thanks for your help on this.
